src/components/tasks/card/index.tsx

```diff
@@ -35,6 +35,8 @@ const Card: FC<Props> = ({
   const iconWidth = '25px';
 
   const cardClassNames = [classNames.card];
+  const dates = localEndsOn == undefined? " " : JSON.stringify(localEndsOn).slice(1, 11);
```

Comment:
How many dates are here?

Reply:
Can this be written a bit more cleanly like:

```js
 const x = !!localEndsOn ? JSON.stringify(localEndsOn).slice(1, 11) : '';
```

First the part of what should happen, not the empty case first.

Reply:
I also don't understand what this JSON stringify is doing? Why do we need it?

Reply:
typeof localEndsOn is object and its gives us "Thu Aug 11 2022 15:37:00 GMT+0530 (India Standard Time)"
We want to extract only date from it.
Example '2022-08-11'

 we will use slice so that we can get selected part of the string but slice can only call on the string or the array.
since localEndOn is Object, so first we will change it into string with he help of JSON.stringify().

Now, We get string and we can use slice method.

When we call slice method on undefined or object then we get Type Error.

This is the main reason, to use JSON.stringify() in the code

Reply:
There are many dates.
because, Every user will have deadline.

Reply:
Yea!,
 const x = !!localEndsOn ? JSON.stringify(localEndsOn).slice(1, 11) : '';
 This is much simple and  working on both "null" and "undefined" .
 Thanks!